unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/ECS7ComponentsPlugin.cs

```diff
@@ -18,14 +19,39 @@ public ECS7ComponentsPlugin(ECSComponentsFactory componentsFactory, IECSComponen
 
     public void RegisterComponents()
     {
+        // Box Shape
         DataStore.i.ecs7.componentsFactory.AddOrReplaceComponent(ComponentID.BOX_SHAPE,
             data => PBBoxShape.Parser.ParseFrom((byte[])data),
             () =>  new ECSBoxShapeComponentHandler());
+        
+        // Plane Shape
+        DataStore.i.ecs7.componentsFactory.AddOrReplaceComponent(ComponentID.PLANE_SHAPE,
```

Comment:
could we use the reference of the factory that is pass through the constructor instead of using the data store?